### PR TITLE
WIP(navbar): Navbar fix

### PR DESCRIPTION
**Pull request by dimeros**
- Bower file correction
- Navbar correction

### CODE REVIEW

File: generators/client-2/templates/src/main/webapp/app/layouts/navbar/_navbar.html [hunk 1]
```diff
@@ -165,17 +158,17 @@
                     </ul>
                 </li>
                 <%_ if (enableTranslation){ _%>
-                <li *ngIf="languages" ngbDropdown class="dropdown pointer">
-                    <a class="dropdown-toggle" ngbDropdownToggle href="javascript:void(0);" *ngIf="languages.length > 1">
+                <li *ngIf="languages" ngbDropdown class="nav-item dropdown pointer" >
+                    <a class="nav-link dropdown-toggle" href="javascript:void(0);" id="languagesnavBarDropdown" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false" *ngIf="languages.length > 1">
```
Comment (deepu105) [inline generators/client-2/templates/src/main/webapp/app/layouts/navbar/_navbar.html]: data-toggle not required as we use ng-bootstrap from js. bs4 is used only for css

File: generators/generator-base.js [hunk 2]
```diff
@@ -46,8 +46,8 @@ Generator.prototype.addElementToMenu = function (routerName, glyphiconName, enab
             file: fullPath,
             needle: 'jhipster-needle-add-element-to-menu',
             splicable: [
-                '<li ui-sref-active="active" >\n' +
-                '                    <a ui-sref="' + routerName + '" ng-click="vm.collapseNavbar()">\n' +
+                '<li class="nav-item" ui-sref-active="active" >\n' +
```
Comment (deepu105) [inline generators/generator-base.js]: this shouldnt be changed at the moment ill do this after entity migration. same for all changes on this file

File: generators/client-2/templates/src/main/webapp/app/layouts/navbar/_navbar.html (hunk 1 above)
Comment (dimeros) [inline generators/client-2/templates/src/main/webapp/app/layouts/navbar/_navbar.html]: Ok I will remove it.

File: generators/generator-base.js (hunk 2 above)
Comment (dimeros) [inline generators/generator-base.js]: OK

File: generators/client-2/templates/src/main/webapp/app/layouts/navbar/_navbar.html [hunk 3]
```diff
@@ -47,47 +40,47 @@
...
                             </a>
                         </li>
                     </ul>
                 </li>
-                <li *ngSwitchCase="true" has-authority="ROLE_ADMIN" ngbDropdown class="dropdown pointer">
-                    <a class="dropdown-toggle" ngbDropdownToggle href="javascript:void(0);" id="admin-menu">
+                <li *ngSwitchCase="true" has-authority="ROLE_ADMIN" ngbDropdown class="nav-item dropdown pointer">
+                    <a class="nav-link dropdown-toggle" ngbDropdownToggle href="javascript:void(0);" id="admin-menu">
                         <span>
                             <span class="glyphicon glyphicon-tower"></span>
```
Comment (dimeros) [inline generators/client-2/templates/src/main/webapp/app/layouts/navbar/_navbar.html]: I didn't find any relevant icon which is the same with the glyphicon-tower

Comment (deepu105) [inline generators/client-2/templates/src/main/webapp/app/layouts/navbar/_navbar.html]: no problem just find something suitable for admin

Comment (dimeros) [inline generators/client-2/templates/src/main/webapp/app/layouts/navbar/_navbar.html]: I will add ths one. http://fontawesome.io/icon/user-plus. No any imagination. :)

**Comment by deepu105** [comment]
closed in favour of #4592

**Pull request closed without merging**